app/service/rest_svc.py

```diff
@@ -234,8 +234,7 @@ async def _build_operation_object(self, access, data):
         name = data.pop('name')
         group = data.pop('group', '')
         planner = await self.get_service('data_svc').locate('planners',
-                                                            match=dict(name=data.pop('planner') if not
-                                                                       data.get('planner') == '' else 'sequential'))
+                                                            match=dict(name=data.pop('planner', 'sequential')))
```

Comment:
This won't work.  The data.pop will check if it's NULL.  UI returns an empty string, not None.

Reply:
Ah okay, my bad. Thanks! I'll revert that